### PR TITLE
[DOCS-80] Describe how to customize pentest reports

### PR DESCRIPTION
**Pull request by mjang-cobalt**
@yogipetkar I've drafted this as the successor to https://cobalt.io/blog/cobalt-platform-deep-dive-customize-your-pentest-reports-per-your-needs, so I think you're the best reviewer here.

You can review the build of this PR in these links:

- [New "Customize Your Report page](https://deploy-preview-103--cobalt-docs.netlify.app/getting-started/customize-report/)
- [Expanded glossary definition for a pentest report](https://deploy-preview-103--cobalt-docs.netlify.app/getting-started/glossary/#pentest-report)
- [Attestation letter definition](https://deploy-preview-103--cobalt-docs.netlify.app/getting-started/glossary/#attestation-letter)

### CODE REVIEW

File: content/en/Getting started/glossary.md [hunk 1]
```diff
@@ -152,14 +152,15 @@ penetration testing programs.
 
 ## Pentest Report
```
Comment (mjang-cobalt) [inline content/en/Getting started/glossary.md:154]: I've changed this definition to reflect what's shown in our UI:
Screen Shot 2022-03-09 at 8 43 12 AM

File: content/en/Getting started/customize-report.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Customize Your Pentest Report"
+linkTitle: "Customize Your Report"
+weight: 310
+description: >
+  You may be able to customize your pentest report.
+---
+
+{{% pageinfo %}}
+This feature may be limited to subscribers with specific [PtaaS Tiers](https://cobaltio.zendesk.com/hc/en-us/articles/4408839706132-Cobalt-PtaaS-Tiers).
```
Comment (mjang-cobalt) [inline content/en/Getting started/customize-report.md:10]: In this case, I'm pointing to the linked Zendesk article as the SSoT.

File: content/en/Getting started/glossary.md (hunk 1 above)
Comment (mjang-cobalt) [reply content/en/Getting started/glossary.md:154]: I have related WIP in [this PR](https://github.com/cobalthq/cobalt-product-public-docs/pull/104/files#diff-7c37a2a0893fd5afb4db9777a0a9996e3c0ce43c80cac2d7c223273683e1eff2)

**Review by yogipetkar — APPROVED** [review]

**Comment by mjang-cobalt** [comment]
Merging based on @yogipetkar 's approval.

File: content/en/Getting started/customize-report.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Customize Your Pentest Report"
+linkTitle: "Customize Your Report"
+weight: 310
+description: >
+  You may be able to customize your pentest report.
```
Comment (TheInstaGrahame) [inline content/en/Getting started/customize-report.md:6]: Main thing I noticed might be a clarification to make: You can't customize the Full/Full+Findings/Customer Letter/Attestation reports. When you customize a report, it appears as a separate item at the bottom of the drop-down.

The other formats are preset formats. Including the "Customized Report" option at the bottom of the list, and mentioning that it's either one of the preset formats or a custom report might fix that.

Screen Shot 2022-04-29 at 10 53 51 AM

File: content/en/Getting started/customize-report.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+   ![Customize a pentest report](/gsg/CustomizePentestReport.png "Select the eye icon to leave a section out of a pentest report.")
+
+1. When you've finished customizing your report, scroll to the top of the page and select **Apply**. 
+
+1. Now you can select **Download** to download your pentest report, as a PDF file, with the
+   changes you configured.
+
+### Pentest Report Types
+
```
Comment (TheInstaGrahame) [inline content/en/Getting started/customize-report.md:43]: A line here like "In addition to the customized report, you can also download one of these preset report formats" might clarify the above.

File: content/en/Getting started/customize-report.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+| Customer Letter               | An executive summary of the pentest. May be used as a certificate of completion. Great for external shareholders. Includes:<br>  <br>- Executive Summary <br>- Methodology |
+| Attestation Report            | Adds the following information to the customer letter: <br> <br>- Pentester user information <br>- An overall list of findings                                                                     |
+| Attestation Letter            | Includes the [executive summary](../what-to-expect/report-contents/executive-summary) as a formal letter, suitable for external stakeholders or customers                                                                     |
+| Full Report                   | Includes the following report sections, beyond attestation:<br>  <br>- Executive Summary <br>- Scope of Work <br>- Methodology <br>- Summary of Findings <br>- Recommendations <br>- Post-Test Remediation     |
+| Full Report + Finding Details | Adds details of every test finding to the full report. Details include:<br>  <br>- Vulnerability Type <br>- Description <br>- Proof of Concept <br>- Severity <br>- Suggested Fix     | 
+
```
Comment (TheInstaGrahame) [inline content/en/Getting started/customize-report.md:51]: For here, I'd suggest adding "| Customized Report (after following the process above) | A customized report with sections shown or hidden per your specifications |"
(Or, y'know, something better)

File: layouts/shortcodes/attestation-letter.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+The Attestation Letter is a one-page report that you can share with external stakeholders 
+such as prospects or customers. We base the letter on our [Executive Summary](/getting-started/what-to-expect/report-contents/#executive-summary).
+You cannot customize an Attestation Letter.
```
Comment (TheInstaGrahame) [inline layouts/shortcodes/attestation-letter.md:3]: Not sure if it's worth mentioning this across all the formats. But if you've got variables for the other formats, it couldn't hurt to make the "you can't customize this" note there too.

**Review by TheInstaGrahame — COMMENTED** [review]
Mainly clarifying something. At least from my playing around, you can either customize or download a preset format. Can't customize any of the pre-set formats.
Screen Shot 2022-04-29 at 10 53 51 AM
"Customized Report" shows up in the drop-down as an option after you make a customized report. The platform seemed to save it when I tabbed away, too.

File: layouts/shortcodes/attestation-letter.md (hunk 6 above)
Comment (mjang) [reply layouts/shortcodes/attestation-letter.md:3]: I don't (yet) have variables for the other formats :)

**Comment by mjang** [comment]
@TheInstaGrahame I've followed up with this PR, to address your comments: https://github.com/cobalthq/cobalt-product-public-docs/pull/127

**Pull request merged**